thakichowdhury/document-templating
# Add pull request template to docs directory

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Semantic version of the package has been incremented if needed (for bug fixes/features)
- [ ] Build (`yarn run build`) was run locally and any changes were pushed
- [ ] Lint (`yarn run lint`) has passed locally and any fixes were made for failures


## Pull request type
 
 
Please check the type of change your PR introduces:
- [x] Other (please describe): developer experience


## What is the current behavior?

- developer has to manually enter pull request description and instructions

## What is the new behavior?

- the pull request description body is prefilled from the pull request template

## Does this introduce a breaking change?



## Testing Instructions

- make a pull request
- see that the description body is prefilled with text from the pull request template

## Other information
